lib/parseMessage.js

```diff
@@ -0,0 +1,11 @@
+module.exports = message => {
+  if(message[0] !== '@' ) return null;
+
+  const pattern = /@(?<command>\W*):?(?<arg>\)
```

Comment:
```suggestion
  const pattern = /@(?<command>\w*):?(?<arg>\w*)\s?(?<text>.*)?/;
  const match = pattern.exec(message);
  if(!match || !match.groups) return null;
```
(If you were attempting to match the patterns from classwork)